exporters/ostream/test/ostream_log_test.cc

```diff
@@ -1,22 +1,29 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-#ifdef ENABLE_LOGS_PREVIEW
-
-#  include "opentelemetry/exporters/ostream/log_record_exporter.h"
-#  include "opentelemetry/exporters/ostream/log_record_exporter_factory.h"
-#  include "opentelemetry/logs/provider.h"
-#  include "opentelemetry/nostd/span.h"
-#  include "opentelemetry/sdk/instrumentationscope/instrumentation_scope.h"
-#  include "opentelemetry/sdk/logs/logger_provider.h"
-#  include "opentelemetry/sdk/logs/read_write_log_record.h"
-#  include "opentelemetry/sdk/logs/simple_log_record_processor.h"
-#  include "opentelemetry/sdk/version/version.h"
-
-#  include <gtest/gtest.h>
-
-#  include <array>
-#  include <iostream>
+#include "opentelemetry/exporters/ostream/log_record_exporter.h"
+#include "opentelemetry/logs/provider.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/instrumentationscope/instrumentation_scope.h"
+#include "opentelemetry/sdk/logs/logger_provider.h"
+#include "opentelemetry/sdk/logs/read_write_log_record.h"
+#include "opentelemetry/sdk/logs/simple_log_record_processor.h"
+#include "opentelemetry/sdk/version/version.h"
+
+#include "opentelemetry/exporters/ostream/log_record_exporter.h"
+#include "opentelemetry/exporters/ostream/log_record_exporter_factory.h"
+#include "opentelemetry/logs/provider.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/instrumentationscope/instrumentation_scope.h"
+#include "opentelemetry/sdk/logs/logger_provider.h"
+#include "opentelemetry/sdk/logs/read_write_log_record.h"
+#include "opentelemetry/sdk/logs/simple_log_record_processor.h"
+#include "opentelemetry/sdk/version/version.h"
```

Comment:
Some of the above headers have been added twice, Can you please check this once.

Reply:
I didn't see any header file is included twice directly. Do you mean some header file in the list is also included indirectly?